components/dashboard/src/teams/TeamSettings.tsx

```diff
@@ -237,22 +221,135 @@ function OrgSettingsForm(props: { org?: OrganizationInfo }) {
+                <div className="flex items-center overflow-hidden" title={props.image}>
+                    <span className="w-5 h-5 mr-1">
+                        <Stack />
+                    </span>
+                    <span className="truncate font-medium text-gray-700 dark:text-gray-200">
+                        {parseDockerImage(props.image).repository}
+                    </span>
+                    &nbsp;&middot;&nbsp;
+                    <span className="flex-none w-16 truncate text-gray-500 dark:text-gray-400">
+                        {parseDockerImage(props.image).tag}
+                    </span>
+                </div>
+                {!props.disabled && (
+                    <Button htmlType="button" type="transparent" className="text-blue-500" onClick={props.onClick}>
+                        Change
+                    </Button>
+                )}
+            </div>
+        </InputField>
+    );
+}
+
+interface OrgDefaultWorkspaceImageModalProps {
+    globalDefaultImage: string | undefined;
+    settings: OrganizationSettings | undefined;
+    onClose: () => void;
+}
+
+function OrgDefaultWorkspaceImageModal(props: OrgDefaultWorkspaceImageModalProps) {
+    const [errorMsg, setErrorMsg] = useState("");
+    const [defaultWorkspaceImage, setDefaultWorkspaceImage] = useState(props.settings?.defaultWorkspaceImage ?? "");
+    const updateTeamSettings = useUpdateOrgSettingsMutation();
+
+    const handleUpdateTeamSettings = useCallback(
+        async (newSettings: Partial<OrganizationSettings>) => {
+            try {
+                await updateTeamSettings.mutateAsync({
+                    ...props.settings,
+                    ...newSettings,
+                });
+                props.onClose();
+            } catch (error) {
+                console.error(error);
+                setErrorMsg(error.message);
+            }
+        },
+        [updateTeamSettings, props],
+    );
+
+    return (
+        <Modal
+            visible
+            closeable
+            onClose={props.onClose}
+            onSubmit={() => handleUpdateTeamSettings({ defaultWorkspaceImage })}
+        >
+            <ModalHeader>Workspace Default Image</ModalHeader>
+            <ModalBody>
+                <Alert type="warning" className="mb-2">
+                    <span className="font-medium">Warning:</span> You are setting a default image for all workspaces
+                    within the organization.
+                </Alert>
+                {errorMsg.length > 0 && (
+                    <Alert type="error" className="mb-2">
+                        {errorMsg}
+                    </Alert>
+                )}
+                <div className="mt-4">
+                    <TextInputField
+                        label="Default Image"
+                        hint="Use any official or custom workspace image from Docker Hub, or any from private container registries that the instance can access."
```

Comment:
> that the instance can access.

I don't think many users would understand what “the instance” is referring to here. Maybe change it to “Gitpod installation” or “Gitpod instance”?

Reply:
I removed that in the latest suggestion to avoid overusing the term, but yeah could be better to bring it back.